lightning/src/ln/functional_tests.rs

```diff
@@ -4088,7 +4088,7 @@ fn test_manager_serialize_deserialize_inconsistent_monitor() {
 }
 
 macro_rules! check_spendable_outputs {
-	($node: expr, $der_idx: expr) => {
+	($node: expr, $der_idx: expr, $chan_signer: expr) => {
```

Comment:
Hmmmm...This is not ok - the point of this check is to check that the user can sign for outputs we notify them of, not that *we* can sign for outputs we notify them of. Making them dig into private fields of the ChannelMonitor to get the chan_signer back and then call methods on it is not ok.

Reply:
Okay changed to `TestKeysInterface::get_descriptor_base_keys()`